test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -124,12 +124,19 @@ public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
         }
         createProxyDatabase(testParam.getDatabaseType());
         if (PipelineEnvTypeEnum.NATIVE == ENV.getItEnvType()) {
+            try {
+                cleanUpPipelineJobs();
+            } catch (final SQLException ex) {
+                throw new RuntimeException(ex);
+            }
             cleanUpDataSource();
         }
         extraSQLCommand = JAXB.unmarshal(Objects.requireNonNull(PipelineBaseE2EIT.class.getClassLoader().getResource(testParam.getScenario())), ExtraSQLCommand.class);
         pipelineWatcher = new PipelineWatcher(containerComposer);
     }
     
+    protected abstract void cleanUpPipelineJobs() throws SQLException;
```

Comment:
1, Could we remove this `cleanUpPipelineJobs` method.

2, Add `cleanUpPipelineJobs` here, transfer rollback and commit SQL in parameters;
then `AbstractMigrationE2EIT` add new method to invoke `cleanUpPipelineJobs`, e.g. `cleanUpMigrationJobs`,
and sub-classes could just invoke `cleanUpMigrationJobs`.